vllm/v1/metrics/stats.py

```diff
@@ -221,6 +221,8 @@ def __init__(self):
         self.num_generation_tokens = 0
         self.num_prompt_tokens = 0
         self.num_preempted_reqs = 0
+        # Num of prompt tokens that have been computed locally.
```

Comment:
Is the naming here a big confusing? By "computed locally" here we mean both computed and locally cached?

If you just tracked `num_external_computed_tokens` and then subtracted it in `_track_iteration_stats()` would that be more clear?